mmv1/products/dlp/JobTrigger.yaml

```diff
@@ -158,6 +175,413 @@ properties:
+                    one detected by the system. If built-in info type is not specified in `info_types` list then the name is
+                    treated as a custom info type.
+                  properties:
+                    - !ruby/object:Api::Type::String
+                      name: 'name'
+                      required: true
+                      description: |
+                        Name of the information type. Either a name of your choosing when creating a CustomInfoType, or one of the names
+                        listed at https://cloud.google.com/dlp/docs/infotypes-reference when specifying a built-in type.
+                    - !ruby/object:Api::Type::String
+                      name: 'version'
+                      description: |
+                        Version of the information type to use. By default, the version is set to stable.
+                - !ruby/object:Api::Type::Enum
+                  name: 'likelihood'
+                  description: |
+                    Likelihood to return for this CustomInfoType. This base value can be altered by a detection rule if the finding meets the criteria
+                    specified by the rule.
+                  values:
+                    - :VERY_UNLIKELY
+                    - :UNLIKELY
+                    - :POSSIBLE
+                    - :LIKELY
+                    - :VERY_LIKELY
+                  default_value: :VERY_LIKELY
+                - !ruby/object:Api::Type::Enum
+                  name: 'exclusionType'
+                  description: |
+                    If set to EXCLUSION_TYPE_EXCLUDE this infoType will not cause a finding to be returned. It still can be used for rules matching.
+                  values:
+                    - :EXCLUSION_TYPE_EXCLUDE
+                - !ruby/object:Api::Type::NestedObject
+                  name: 'regex'
+                  description: Regular expression which defines the rule.
+                  immutable: true
+                  properties:
+                    - !ruby/object:Api::Type::String
+                      name: 'pattern'
+                      required: true
+                      description: |
+                        Pattern defining the regular expression.
+                        Its syntax (https://github.com/google/re2/wiki/Syntax) can be found under the google/re2 repository on GitHub.
+                    - !ruby/object:Api::Type::Array
+                      name: 'groupIndexes'
+                      description: |
+                        The index of the submatch to extract as findings. When not specified, the entire match is returned. No more than 3 may be included.
+                      item_type: Api::Type::Integer
+                - !ruby/object:Api::Type::NestedObject
+                  name: 'dictionary'
+                  description: Dictionary which defines the rule.
+                  immutable: true
```

Comment:
Is `immutable: true` is required here?

Reply:
I've checked this field and it runs fine without `ForceNew`. Hence, removing `immutable:true`.